Mathlib/Data/List/Sort.lean

```diff
@@ -202,6 +225,24 @@ alias ⟨_, _root_.Antitone.ofFn_sorted⟩ := sorted_ge_ofFn_iff
 
 end Monotone
 
+lemma Sorted.filterMap {α β : Type*} [DecidableEq α] [DecidableEq β]
```

Comment:
```suggestion
lemma filterMap {α β : Type*} [DecidableEq α] [DecidableEq β]
```

Reply:
Actually shouldn't this stay as `List.Sorted.filterMap` to stay consisted with [List.Pairwise.map](https://leanprover-community.github.io/mathlib4_docs/Init/Data/List/Pairwise.html#List.Pairwise.map)?

Reply:
Ah, I thought the `Sorted` namespace was open already here